app/views/Home/index.tsx

```diff
@@ -35,49 +36,60 @@ import RecentActivity from './RecentActivity';
 
 import styles from './styles.css';
 
-interface RecentProjectItemProps {
-    projectId: number;
-    title: string;
-    isPrivate: boolean;
-    description?: string;
-    projectOwnerName: string;
-    analysisFrameworkTitle?: string;
-    startDate?: string;
-    endDate?: string;
-    totalUsers: number;
-    totalSources: number;
-    totalSourcesTagged: number;
-    totalSourcesValidated: number;
-    entriesActivity: CountTimeSeries[];
-    recentlyActive: UserActivityStat[];
-}
+const RECENT_PROJECTS = gql`
+    query RecentProjects{
+      recentProjects {
+        id
+        title
+        isPrivate
+        description
+        startDate
+        endDate
+        analysisFramework {
+          id
+          title
+        }
+        createdBy {
+          displayName
+        }
```

Comment:
Always pull `id` even if it is not used.
This helps apollo to udpate the normalized cache